salmon.py

```diff
@@ -12,39 +12,51 @@
 class SalmonDataset(Dataset):
     def __init__(self, salmon_path, part, load_audio=True):
         self.data = []
-        self.salmon_path = Path(salmon_path)
         self.load_audio = load_audio
-        dir_path = self.salmon_path / part
-        paths = list(dir_path.glob("*.wav"))
+        self.HF = salmon_path is None  # set Hugging Face flag if data path hasn't specified
```

Comment:
I prefer to follow convention of using lower case letters for variables self.use_hugging_face?